src/walletSdk/horizon/Account.ts

```diff
@@ -0,0 +1,52 @@
+import { Keypair, Transaction, FeeBumpTransaction } from "stellar-sdk";
+
+class AccountKeypair {
+  keypair: Keypair;
+  constructor(keypair: Keypair) {
+    this.keypair = keypair;
+  }
+
+  get publicKey(): string {
+    return this.keypair.publicKey();
+  }
+
+  toString(): string {
+    return `${this.constructor.name}(address=${this.publicKey})`;
+  }
+}
+
+export class PublicKeypair extends AccountKeypair {
+  keypair: Keypair;
+  constructor(keypair: Keypair) {
+    super(keypair);
+  }
+
+  static fromPublicKey = (str: string) => {
```

Comment:
could we add return type here as well?

sorry for being annoying about the typing, I'm just trying to keep consistency throughout the codebase :sweat_smile:

maybe we should have a task dedicated to reviewing and adding types everywhere

Reply:
not a problem glad we're catching / adding these 😄
https://github.com/stellar/typescript-wallet-sdk/pull/21/commits/f4dbf01c448df491da717687e4591e393b835d29